src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,42 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
```

Comment:
FYI, you can also make this like
`Animal.alive = []`
outside of class :)

But this is a better way, this task just has eslint configured the way that it will say it's wrong